src/CameraProps.ts

```diff
@@ -161,6 +161,15 @@ export interface CameraProps extends ViewProps {
    */
   orientation?: 'portrait' | 'portraitUpsideDown' | 'landscapeLeft' | 'landscapeRight';
 
+  /**
+   * Specifies the compression quality of the photo. The value ranges from `1` (lowest quality) to `100` (highest quality).
+   * 
+   * @platform Android
+   * @default 0 - not set, in which case it uses [`95`](https://developer.android.com/reference/kotlin/androidx/camera/core/ImageCapture.Builder#setjpegquality)
+   * 
+   */
+  jpegQuality?: number;
+
```

Comment:
I kinda don't like how we have two separate props for this. One is for the `<Camera>`, one is for the options in `takePhoto(...)`. Either we do both to `<Camera>` and just use that for iOS as well, or we find a way to get the one on Android into `takePhoto(...)` too (better in my opinion)

Do you think it's possible we can move `jpegQuality` to `takePhoto(...)` for both platforms?

Reply:
yes, having separate props is very painful to me as well.

the big issue is that on android, the jpeg quality setting is part of the [`ImageCapture.Builder` class](https://developer.android.com/reference/androidx/camera/core/ImageCapture.Builder#setJpegQuality(int)) and this is set up while initializing the camera, way before we do a takePicture.

on the other hand, on ios, the only (seemingly) way to set jpeg quality is when taking the picture, at least from the research i've done so far.

also, not sure how the previous react-native-camera implemented it, but i think you're using a different camera class on android right ? so it wouldn't help to explore that